test/extensions/filters/http/lua/lua_filter_test.cc

```diff
@@ -193,7 +193,7 @@ class LuaHttpFilterTest : public testing::Test {
     function envoy_on_request(request_handle)
       request_handle:logTrace(request_handle:headers():get(":path"))
 
-      if request_handle:body() ~= nil then
+      if request_handle:body():length() ~= 0 then
```

Comment:
This requires Release Notes since it breaks API. And we should introduce a runtime feature flag for this.

Reply:
@dio
I am new to envoy. What runtime feature flag should we use and should we turn on it by default?

Reply:
Sorry for not being clear. The general info of this can be found in https://github.com/envoyproxy/envoy/blob/master/CONTRIBUTING.md#runtime-guarding. 

You can add one here: https://github.com/envoyproxy/envoy/blob/e90d674fd2eddd883481011bf580fbb7080f535b/source/common/runtime/runtime_features.cc#L54

For example: "envoy.reloadable_features.lua_always_wrap_body" (or similar, WARNING: I just made up this name). And make sure we test both cases (when the flag is on and off).

Reply:
@dio 
Resolved.

Reply:
Sorry that I didn't entirely clear here. This requires a "Release Notes" entry. https://github.com/envoyproxy/envoy/blob/master/docs/root/version_history/current.rst. I think this should be included as an entry in "Minor Behavior Changes".

Reply:
@dio 
Got it. Added in the new commit.